lib/spreedly/environment.rb

```diff
@@ -191,6 +191,13 @@ def add_payment_token(doc, payment_method_token, options = {})
             #{'<test_card_number>4111111111111111</test_card_number>' if options[:test_mode]}
           </google_pay>
         XML
+      elsif options[:payment_method] == :apple_pay
+        doc << <<~XML
+          <apple_pay>
+            <payment_data><![CDATA[#{payment_method_token}]]></payment_data>
+            #{'<test_card_number>4111111111111111</test_card_number>' if options[:test_mode]}
+          </apple_pay>
+        XML
```

Comment:
We can probably do something fancier here in which we use the value of the options[:payment_method] as the value for <apple_pay> or <google_pay>

Example:
```
doc << <<~XML
  <#{options[:paymet_method]}
    ....
  </#{options[:paymet_method]}
 ```